sdk/cosmos/azure-cosmos/azure/cosmos/aio/_cosmos_client.py

```diff
@@ -250,6 +251,33 @@ def from_connection_string(
             **kwargs
         )
 
+    @overload
+    async def create_database(
+            self,
+            id: str,
+            *,
+            offer_throughput: Optional[Union[int, ThroughputProperties]] = None,
+            initial_headers: Optional[Dict[str, str]] = None,
+            throughput_bucket: Optional[int] = None,
+            return_properties: Literal[False],
```

Comment:
The `False` scenario is the default - so we can add that here. It's only a required parameter in the `True` scenario.
```suggestion
            return_properties: Literal[False] = False,
```